admin/fix-ubuntu-build

JacksonMaxfield opened the pull request:
<!--
  Thank you for submitting a pull request!

  ⚠️⚠️ Please do the following before submitting: ⚠️⚠️

  - Read the CONTRIBUTING.md guide and make sure you've followed all the steps given.
  - Ensure that the code is up-to-date with the `main` branch.
  - Provide or update documentation for any feature added by your pull request.
  - Provide relevant tests for your feature or bug fix.

  ❗️ Also: ❗️

  Please name your pull request {development-type}/{short-description}.
  For example: feature/read-tiff-files
-->

### Link to Relevant Issue

Fixing Ubuntu Builds for #89 

### Description of Changes

_Include a description of the proposed changes._

I believe the `ffmpeg` is the package that is failing not graphviz due to the doc tests passing and they install graphviz. To resolve I am adding `--fix-missing` to all `apt-get` calls.

isaacna commented:
Sorry not sure if you saw the last few commits I added, but I added the fix in my [PR]( https://github.com/CouncilDataProject/cdp-backend/commit/540392eae5194c5cad5105d6a6a0dba8d88aaa0d#diff-6f7a95041979238479eb79ffa787510811d6a9bd71382b8e542a295cfb2fe15aR34-R35). It was failing on my PR since the github actions bot wasn't running with the changes to it in the PR, but now that it's merged in the [build succeeds on Ubuntu](https://github.com/CouncilDataProject/cdp-backend/runs/3390470096).

JacksonMaxfield commented:
> Sorry not sure if you saw the last few commits I added, but I added the fix in my [PR](https://github.com/CouncilDataProject/cdp-backend/commit/540392eae5194c5cad5105d6a6a0dba8d88aaa0d#diff-6f7a95041979238479eb79ffa787510811d6a9bd71382b8e542a295cfb2fe15aR34-R35). It was failing on my PR since the github actions bot wasn't running with the changes to it in the PR, but now that it's merged in the [build succeeds on Ubuntu](https://github.com/CouncilDataProject/cdp-backend/runs/3390470096).

Oh lol. Missed that. But no worries this applies the fix to all the various CI stuff we have.

isaacna reviewed (approved):
Good call preemptively adding `--fix-missing` everywhere
